Mathlib/Algebra/Group/Defs.lean

```diff
@@ -898,6 +898,15 @@ theorem exists_zpow_surjective (G : Type*) [Pow G ℤ] [IsCyclic G] :
     ∃ g : G, Function.Surjective (g ^ · : ℤ → G) :=
   IsCyclic.exists_zpow_surjective
 
+lemma IsCyclic.push_ofSurjective {G H : Type*} [Pow G ℤ] [Pow H ℤ] (h : IsCyclic G)
```

Comment:
This seems to somwhat duplicate [`isCyclic_of_surjective`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/GroupTheory/SpecificGroups/Cyclic.html#isCyclic_of_surjective).

Reply:
Yes, I've replaced it with `isCyclic_of_surjective`, thanks.